packages/dds/tree/src/feature-libraries/default-schema/schemaChecker.ts

```diff
@@ -28,6 +29,15 @@ export const enum SchemaValidationErrors {
 	UnknownError,
 }
 
+/**
+ * Throws a UsageError if maybeError indicates a tree is out of schema.
+ */
+export function inSchemaOrThrow(maybeError: SchemaValidationErrors): void {
```

Comment:
this utility is used by users of the validator in this file, so putting it here makes sense and increases who can use it.